### PR TITLE
SIP providers: add a template for the OpenIP and voipms operator

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

File: plugins/sip/openipv1/endpoint/variables.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                {"language": "fr_CA", "value": "Mot de passe"},
+                {"language": "fr_FR", "value": "Mot de passe"},
+                {"language": "en_US", "value": "Password"}
+            ],
+            "required": true
+        },
+        {
+            "name": "wazo_uc_stack_ip_address",
+            "label": [
+                {"language": "fr_CA", "value": "Adress IP ou hostname du stack Wazo UC"},
```
Comment (manuquentin) [inline plugins/sip/openipv1/endpoint/variables.json:32]: `Adresse` (same line below)

File: plugins/sip/openipv1/endpoint/body.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "endpoint_section_options": [
+        ["identify_by", "header,auth_username,username"],
+        ["allow", "!all,alaw"]
+    ],
+    "registration_section_options": [
+        ["contact_user", "{{ username }}"],
+        ["client_uri", "sip:{{ username }}@voip.myopenip.fr:5060"],
+        ["expiration", "60"],
+        ["line", "yes"],
+        ["server_uri", "sip:voip.myopenip.fr:5060"]
```
Comment (chartrandf) [inline plugins/sip/openipv1/endpoint/body.json:15]: Is it required to add `:5060`? I know that the UI may remove it because it's the default port.

Comment (sboily) [reply plugins/sip/openipv1/endpoint/body.json:15]: If we use 5070, how do it works?

Comment (chartrandf) [reply plugins/sip/openipv1/endpoint/body.json:15]: Don't know Sylvain if your question is for me or Pascal. I'm just asking if we should add port only when it's not `5060`, like we do in Portal right now. So if we have 5070, we can add it in the `server_uri`, `client_uri`, etc.

**Pull request merged**